package.json

```diff
@@ -7,8 +7,11 @@
    "module": "./dist/esm/index",
    "scripts": {
       "prepare": "grunt build",
-      "test": "check-node-version --npm 6.13.4 && TS_NODE_PROJECT='tests/tsconfig.json' TS_NODE_FILES=true nyc mocha --opts ./.mocha.opts",
-      "commitlint": "commitlint --from 86324da"
+      "test": "check-node-version --npm 6.14.12 && NODE_OPTIONS=--unhandled-rejections=warn TS_NODE_PROJECT='tests/tsconfig.json' TS_NODE_FILES=true nyc mocha --opts ./.mocha.opts",
```

Comment:
Node 15 changes how unhandled promise rejections are handled, so when pipelines with current Node LTS and later ran, tests would fail when running tests that evaluate error states. 

Setting handling of rejections back to `warn` allows the tests to pass without significant refactors.

See: https://nodejs.org/en/blog/release/v15.0.0/#throw-on-unhandled-rejections-33021

Reply:
Genius!